custom_components/octopus_intelligent/graphql_client.py

```diff
@@ -191,7 +191,7 @@ async def __async_cancel_boost_charge(self, session, account_id: str):
     result = await session.execute(query, variable_values=params, operation_name="deleteBoostCharge")
     return result['deleteBoostCharge']
 
-  async def __async_get_accounts(session):
+  async def __async_get_accounts(self, session):
```

Comment:
This change is another git cherry-pick from PR #38. If that PR is merged first, this change disappears from here.